modules/developer-lightspeed/proc-installing-and-configuring-lightspeed.adoc

```diff
@@ -23,70 +23,73 @@ You must install and configure both the {ls-short} and the {rcs-short} sidecar c
 kind: ConfigMap
 apiVersion: v1
 metadata:
- name: rcsconfig
- namespace: _<namespace>_ # Enter your namespace (For example, `rhdh`)
-data:
- rcsconfig.yaml: |
-  llm_providers:
-    # Start: Do not remove this block
-    - name: dummy
-      type: openai
-      url: https://dummy.com
-      models:
-        - name: dummymodel
-      # End: Do not remove this block
-  ols_config:
-    user_data_collection:
-      log_level: "DEBUG"
-      feedback_disabled: false
-      feedback_storage: "/tmp/feedback"
-    reference_content:
-      product_docs_index_path: "./vector_db/rhdh_product_docs/1.7"
-      product_docs_index_id: rhdh-product-docs-1_7
-      embeddings_model_path: "./embeddings_model"
-    conversation_cache:
-      type: memory
-      memory:
-        max_entries: 1000
-    logging_config:
-      app_log_level: info
-      lib_log_level: warning
-      uvicorn_log_level: info
-      suppress_metrics_in_log: false
-      suppress_auth_checks_warning_in_log: false
-    authentication_config:
-      module: "noop"
-    default_provider: dummy
-    default_model: dummymodel
-    query_validation_method: disabled
-  dev_config:
-    enable_dev_ui: false
-    disable_auth: false
-    disable_tls: true
-    enable_system_prompt_override: true
-  user_data_collector_config:
-    user_agent: "example-user-agent"
-    ingress_url: "https://example.ingress.com/upload"
-----
-+
-[IMPORTANT]
-====
-Do not remove the block in the `llm_providers` section. This requirement is crucial when working with {ls-short} because of limitations discovered in Road Core. If you decide to use an alternative LLM provider, you should refer to additional information in the guide. For more information, see {developer-lightspeed-link}#proc-changing-your-llm-provider_customizing-developer-lightspeed[Changing your LLM provider].
-====
-.. Optional: Configure the number of workers that scale the REST API by specifying the following example to the `ols_config.max_workers` parameter in the {rcs-short} ConfigMap.
+ name: Lightspeed Core Service (LCS)
+service:
+  host: 0.0.0.0
+  port: 8080
+  auth_enabled: false
+  workers: 1
+  color_log: true
+  access_log: true
+llama_stack:
+  use_as_library_client: false
+  url: http://localhost:8321
+user_data_collection:
+  feedback_enabled: true
+  feedback_storage: "/tmp/data/feedback"
+  transcripts_enabled: true
+  transcripts_storage: "/tmp/data/transcripts"
+authentication:
+  module: "noop"
+conversation_cache:
+  type: "sqlite"
+  sqlite:
+    db_path: "./lcs_cache.db"
+mcp_servers:
+  - name: mcp::backstage
+    provider_id: model-context-protocol
+    url: https://rhdh-mcp-proxy-apicast-production.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com:443/api/mcp-actions/v1
```

Comment:
lets remove this URL and specify something along the lines from the MCP documentation that you have which is being reviewed by John